templates/bridgeserver2.yaml

```diff
@@ -385,6 +387,9 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: cuimc.prod.password
           Value: !Ref CuimcProdPassword
+       - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: participant-file.bucket
+          Value: !Ref ParticipantFileBucket
```

Comment:
CFN will create a bucket with a random name then this !Ref will return that bucket name.  Therefore you don't need to pass in the bucket name.